example/dj/apps/app/tests/serializer.py

```diff
@@ -49,3 +53,34 @@ def test_direct_serialization_to_csv_should_create_columns_according_to_required
             serialize(data, converter_name='csv', requested_fieldset=('a', 'b', 'c')),
             '\ufeff"A";"B";"C"\r\n"1";"2";""\r\n'
         )
+
+    def test_direct_serialization_to_python_format_should_return_only_base_python_data(self):
+        now_value = datetime.now().replace(microsecond=0)
```

Comment:
Are you skipping the "microsecond logic" intentionally?

Reply:
only for tests

Reply:
and yes. Because check microseconds is very complicated. And if I do it I will have to implement the helper which is testeing

Reply:
you would have to duplicate the logic from the serializer which would ruin the point of the test, right?

Reply:
yes exactly